src/com/google/enterprise/adaptor/SensitiveValueCodec.java

```diff
@@ -268,11 +271,27 @@ String getPrefix() {
    * -Dserver.keyAlias=adaptor \
    * -Dserver.secure=true
    * </pre>
+   *
+   * Example command line to run (passing the sensitive value on the command
+   * line):
+   * <pre>
+   * java \
+   * -Djavax.net.ssl.keyStore=keys.jks \
+   * -Djavax.net.ssl.keyStoreType=jks \
+   * -Djavax.net.ssl.keyStorePassword=changeit \
+   * -classpath 'adaptor-20130612-withlib.jar' \
+   * com.google.enterprise.adaptor.SensitiveValueCodec \
+   * -DsecurityLevel=ENCRYPTED \
+   * -Dserver.keyAlias=adaptor \
+   * -Dserver.secure=true
+   * 'sensitive value'
```

Comment:
Right, I forgot to update this. I now removed the 2nd example command line, as the sensitive value is again always prompted in some way. Instead I added an info text explaining the new parameter and pointing out the potential risk when using this parameter. 

Oh, and I renamed the parameter from "quiet" to "stdin" as this name is way more applicable in my opinion. I found the same parameter to be used in a version of the passwd command (http://linux.die.net/man/1/passwd). So I believe this is a better name. If you disagree, I can also change the name back to "quiet".